core-api/src/main/resources/definitions/exposed-services.json

```diff
@@ -760,6 +760,66 @@
       "visibleForDatahub": true,
       "minVersion": "7.2.14"
     },
+    {
+      "name": "SSB-SSE-UI",
+      "displayName": "Streaming SQL ENGINE",
+      "serviceName": "STREAMING_SQL_ENGINE",
+      "knoxService": "SSB-SSE-UI",
+      "knoxUrl": "/ssb-sse-ui/",
+      "ssoSupported": true,
+      "port": 18121,
+      "tlsPort": 18121,
+      "apiOnly": false,
+      "apiIncluded": false,
+      "visibleForDatalake": false,
+      "visibleForDatahub": true,
+      "minVersion": "7.2.16"
+    },
+    {
+      "name": "SSB-SSE-WS",
+      "displayName": "Streaming SQL Engine WebSocket",
+      "serviceName": "STREAMING_SQL_ENGINE",
+      "knoxService": "SSB-SSE-WS",
+      "knoxUrl": "/ssb-sse-ws/",
+      "ssoSupported": true,
+      "port": 18121,
+      "tlsPort": 18121,
+      "apiOnly": false,
+      "apiIncluded": false,
+      "visibleForDatalake": false,
+      "visibleForDatahub": false,
+      "minVersion": "7.2.16"
+    },
```

Comment:
These should be mutually exclusive with `SSB-SSC-*` services so please set the max version to `7.2.15` for these two: https://github.com/hortonworks/cloudbreak/blob/520c49bf5814324e21a8709edb8eda5d23ca8a9a/core-api/src/main/resources/definitions/exposed-services.json#L703-L732